metadata/buckets.go

```diff
@@ -36,7 +36,7 @@ var (
 	bucketKeyObjectSnapshots  = []byte("snapshots")  // stores snapshot references
 	bucketKeyObjectContent    = []byte("content")    // stores content references
 	bucketKeyObjectBlob       = []byte("blob")       // stores content links
-	bucketKeyObjectIngest     = []byte("ingest")     // stores ingest links
+	bucketKeyObjectIngests    = []byte("ingests")    // stores ingest objects
```

Comment:
is this backwards compatible?

Reply:
I should have called this change more out in the description.  I had to change the key name to make it backwards compatible, the old key is below and I have called it out as deprecated. The migration script will take care of moving from the old key to the new key. This was trying to be safe in case containerd is downgraded, the ingests will just be gone rather than an unexpected type.